### PR TITLE
global merge (reconfiguring vars-structure)

### PR DESCRIPTION
**Pull request by naumvd95**
https://mirantis.jira.com/browse/FPL-2688
https://mirantis.jira.com/browse/FPL-2685
jjb test passed on all jobs

### CODE REVIEW

File: servers/partner-ci/9.0/contrail.yaml [hunk 1]
```diff
@@ -2,32 +2,25 @@
...
-          default: 'http://mirror.seed-cz1.fuel-infra.org'
+      - text:
+          name: SPECIFIC_PROPS
+          default: |
+            KVM_USE=True
+            NODE_VOLUME_SIZE=512
+            CONTRAIL_VERSION=5.0
+            VSRX_ORIGINAL_IMAGE_PATH=/storage/contrail/vSRX.400.img
+            VSRX_TARGET_IMAGE_PATH=/var/lib/libvirt/images/vSRX.400.img
+            VSRX_TEMPLATE_PATH=/storage/contrail/vSRX_400.template'
```
Comment (rkhozinov) [inline servers/partner-ci/9.0/contrail.yaml]: `VSRX_TEMPLATE_PATH=/storage/contrail/vSRX_400.template'` <<error

File: servers/partner-ci/9.0/contrail.yaml [hunk 2]
```diff
@@ -448,20 +441,23 @@
...
     builders:
       - get-refs
+      - shell: |
+          if [ ! -z "${{COMMON_PROPS}}" ]; then
+            echo > common.properties
+            for param in ${{COMMON_PROPS}}; do
+              echo $param >> common.properties
+            done
+          fi
+          if [ ! -z "${{SPECIFIC_PROPS}}" ]; then
```
Comment (rkhozinov) [inline servers/partner-ci/9.0/contrail.yaml]: Seems like logic mistake:
If `COMMON_PROPS` doesn't exist, then common.properties won't be erased, and then will be injected with invalid properties. Please clean `*.properties` in for each case

File: servers/partner-ci/9.0/contrail.yaml [hunk 3]
```diff
@@ -448,20 +441,23 @@
       - test-property
     builders:
       - get-refs
+      - shell: |
+          if [ ! -z "${{COMMON_PROPS}}" ]; then
```
Comment (rkhozinov) [inline servers/partner-ci/9.0/contrail.yaml]: the same as below

File: servers/partner-ci/9.0/contrail.yaml (hunk 1 above)
Comment (rkhozinov) [reply servers/partner-ci/9.0/contrail.yaml]: fixed

File: servers/partner-ci/9.0/contrail.yaml (hunk 2 above)
Comment (rkhozinov) [reply servers/partner-ci/9.0/contrail.yaml]: fixed

File: servers/partner-ci/9.0/contrail.yaml [hunk 4]
```diff
@@ -513,42 +497,13 @@
...
-                  ENABLE_MOS_CENTOS_SECURITY=true
-                  ENABLE_MOS_UBUNTU_OS=true
-                  ENABLE_MOS_UBUNTU_PROPOSED=$ENABLE_PROPOSED
-                  ENABLE_MOS_UBUNTU_UPDATES=true
-                  ENABLE_MOS_UBUNTU_HOLDBACK=true
-                  ENABLE_MOS_UBUNTU_HOTFIX=false
-                  ENABLE_MOS_UBUNTU_SECURITY=true
-      - conditional-step:
-          condition-kind: shell
-          condition-command: |
```
Comment (naumvd95) [inline servers/partner-ci/9.0/contrail.yaml]: why remove?

**Comment by rkhozinov** [comment]
@naumvd95  I've merged #38 to that branch, so please check it also

**Review by rkhozinov — APPROVED** [review]

**Pull request merged**